docs/_includes/api/replication.html

```diff
@@ -14,11 +14,17 @@
 
 * `options.live`: If `true`, starts subscribing to future changes in the `source` database and continue replicating them.
 * `options.retry`: If `true` will attempt to retry replications in the case of failure (due to being offline), using a backoff algorithm that retries at longer and longer intervals until a connection is re-established. Only applicable if `options.live` is also `true`.
-* `options.filter`: Reference a filter function from a design document to selectively get updates.
-* `options.query_params`: Query params sent to the filter function.
-* `options.doc_ids`: Only replicate docs with these ids (array of strings).
 * `options.since`: Replicate changes after the given sequence number.
-* `options.create_target`: Create target database if it does not exist. Only for server replications.
```

Comment:
`create_target` is not needed anymore, because we don't have server-initiated replication anymore